packages/shared/src/models/user.ts

```diff
@@ -1,8 +1,10 @@
 export type User = {
   id: number;
-  firstName: string;
-  lastName: string;
+  githubId: number;
+  firstName: string | null;
```

Comment:
Maybe we can create `fullName` field instead of `firstName` + `lastName` here and use `displayName` provided by GitHub as an initial value?